elyra/lsp/yarn.lock

```diff
@@ -0,0 +1,65 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```

Comment:
a bit weird to have npm related files in the python dirs, possible we can find another place for these?

Reply:
I agree, it is a bit awkward, is there a place we can put them where they will still be packed up in the python stuff?